Gifski/Utilities.swift

```diff
@@ -374,11 +378,30 @@ extension AVAssetImageGenerator {
 						break
 					}
 
-					// macOS 11 started throwing “decode failed” error for some frames in screen recordings. As a workaround, we ignore these. We throw an error if more than 50% of the frames could not be decoded.
+					// macOS 11 (still an issue in macOS 11.1) started throwing “decode failed” error for some frames in screen recordings. As a workaround, we ignore these. We throw an error if more than 50% of the frames could not be decoded.
 					if error.code == .decodeFailed {
-						decodeFailureFrameCount += 1
-						totalCount -= 1
-						Crashlytics.recordNonFatalError(error: error, userInfo: ["requestedTime": requestedTime.seconds])
+						var newActualTime = CMTime.zero
+						if let image = try? self.copyCGImage(at: requestedTime, actualTime: &newActualTime) {
```

Comment:
This seems to funnily succeed even when `generateCGImagesAsynchronously` failed at the same requested time 🤷🏻‍♂️